sam-app/lambda_functions/template.yaml

```diff
@@ -232,7 +236,7 @@ Conditions:
 Globals:
     Function:
         Timeout: 6
-        Runtime: python3.7
+        Runtime: python3.10
         CodeUri: ./sam-app-deployment-artifact.zip
```

Comment:
I think you also need to update the lambda-layers.zip file with the new 3.10 dependencies zip file generated locally by the release target

Reply:
Done